package.json

```diff
@@ -31,7 +31,7 @@
   },
   "scripts": {
     "build": "webpack",
-    "test": "./node_modules/karma/bin/karma start tests/karma.conf.js",
+    "test": "karma start tests/karma.conf.js",
```

Comment:
I think we had the path to `karma` here so that a contributor didn't need to have karma installed globally.  We probably want to revert this.